### PR TITLE
Fix for the list element order mismatch issue on the TypeList attributes

### PR DESCRIPTION
**Pull request by sajagana**
Note:

Fix for the list element order mismatch issue on the TypeList attributes also updated a few examples.

### CODE REVIEW

File: examples/ospf_timers/main.tf [hunk 1]
```diff
@@ -14,13 +14,17 @@ provider "aci" {
   insecure = true
 }
 
+resource "aci_tenant" "tenentcheck" {
+  name = "tenentcheck"
+}
+
 resource "aci_ospf_timers" "example" {
   tenant_dn           = aci_tenant.tenentcheck.id
```
Comment (akinross) [inline examples/ospf_timers/main.tf:22]:
```suggestion
  tenant_dn           = aci_tenant.footenant.id
```

**Review by akinross — CHANGES_REQUESTED** [review]

**Review by akinross — APPROVED** [review]
LGTM

File: website/docs/r/cloud_subnet.html.markdown [hunk 2]
```diff
@@ -45,7 +45,7 @@ resource "aci_cloud_subnet" "foocloud_subnet" {
 * `description` - (Optional) Description of the Cloud Subnet object.
 * `annotation` - (Optional) Annotation of the Cloud Subnet object.
 * `name_alias` - (Optional) Name alias of the Cloud Subnet object.
-* `scope` - (Optional) List of domain applicable to the capability. Allowed values are "public", "private" and "shared". Default is ["private"]. Use the sorted scope list to handle identical changes.
+* `scope` - (Optional) List of domain applicable to the capability. Allowed values are "public", "private" and "shared". Default is ["private"].
```
Comment (shrsr) [inline website/docs/r/cloud_subnet.html.markdown:48]: List of domains applicable to the capability.

**Review by shrsr — CHANGES_REQUESTED** [review]

**Review by shrsr — APPROVED** [review]
LGTM

**Review by anvitha-jain — APPROVED** [review]
LGTM

**Pull request merged**